python/ray/tests/test_actor.py

```diff
@@ -2074,6 +2074,28 @@ def method(self):
     ray.get(results)
 
 
+@pytest.mark.parametrize(
+    "ray_start_cluster_head", [{
+        "num_cpus": 5,
+        "object_store_memory": 10**7
+    }],
+    indirect=True)
+@pytest.mark.parametrize("num_actors", [1, 2, 5])
+def test_parallel_actor_fill_plasma_retry(ray_start_cluster_head, num_actors):
+    @ray.remote
+    class LargeMemoryActor(object):
+        def some_expensive_task(self):
+            return np.zeros(10**7 // 2, dtype=np.uint8)
+
+    i = 0
+    actors = [LargeMemoryActor.remote() for _ in range(num_actors)]
+    for _ in range(30):
+        pending = [a.some_expensive_task.remote() for a in actors]
+        while pending:
+            [done], pending = ray.wait(pending, num_returns=1)
+            i += 1
```

Comment:
Can we add a test the exception is raised after the timeout?

That's actually the more important case.